content/docs/deployment/csmoperator/_index.md

```diff
@@ -0,0 +1,217 @@
+
+## Supported Platforms
+Dell CSM Operator has been tested and qualified on Upstream Kubernetes and OpenShift. Supported versions are listed below.
+
+| Kubernetes Version   | OpenShift Version     |
+| -------------------- | --------------------- |
+| 1.21, 1.22, 1.23     | 4.8, 4.8 EUS, 4.9     |
+
+## Installation
+**Dell CSM Operator can be installed manually or via OLM (Operator Lifecycle Manager).**
+
+### Manual Installation
+
+#### Operator Installation on an environment without OLM
+1. Clone the [Dell CSM Operator repository](https://github.com/dell/csm-operator).
+2. Run `bash scripts/install.sh` to install the operator.
+>NOTE: Dell CSM Operator will install to the 'dell-csm-operator' namespace by default.
+
+{{< imgproc install.jpg Resize "2500x" >}}{{< /imgproc >}}
+
+3. Run the command `kubectl get pods -n dell-csm-operator` to validate the installation. If completed successfully, you should be able to see the operator-related pod in the 'dell-csm-operator' namespace.
+
+{{< imgproc install_pods.jpg Resize "2500x" >}}{{< /imgproc >}}
+   
+#### Operator Installation on an environment with OLM
+1. Clone the [Dell CSM Operator repository](https://github.com/dell/csm-operator).
+2. Run `bash scripts/install_olm.sh` to install the operator.
+>NOTE: Dell CSM Operator will install to the 'test-csm-operator-olm' namespace by default.
+
+{{< imgproc install_olm.jpg Resize "2500x" >}}{{< /imgproc >}}
+
+3. Once installation completes, run the command `kubectl get pods -n test-csm-operator-olm` to validate the installation. If completed successfully, 
+   you should be able to see the operator-related pods in the `test-csm-operator-olm` namespace. Also, you can see a `dell-csm-operator` CSV with the status 'Succeeded'
+   created in the namespace `test-csm-operator-olm`.
+   
+{{< imgproc install_olm_pods.jpg Resize "2500x" >}}{{< /imgproc >}}
+
+### Installation Using Operator Lifecycle Manager
+`dell-csm-operator` can be installed using Operator Lifecycle Manager (OLM) on upstream Kubernetes clusters & Red Hat OpenShift Clusters.  
+The installation process involves the creation of a `Subscription` object either via the _OperatorHub_ UI or using `kubectl/oc`. While creating the `Subscription` you can set the Approval strategy for the `InstallPlan` for the Operator to - 
+* _Automatic_ - If you want the Operator to be automatically installed or upgraded (once an upgrade becomes available)
+* _Manual_ - If you want a Cluster Administrator to manually review and approve the `InstallPlan` for installation/upgrades
+
+**NOTE**: The recommended version of OLM for upstream Kubernetes is **`v0.18.2`**.
+
+##### Upstream Kubernetes
+- For installing via OperatorHub.io on Kubernetes, go to the [OperatorHub page](../../partners/operator/).
```

Comment:
@rensyct these links to OperatorHub and OpenShift page (line 59) return 404 errors. Should these links be removed since we can't get screenshots of the CSM Operator from these web UIs?

Reply:
Done